components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -634,4 +718,32 @@ private void deleteDELETEOperationsTask() {
 
         }
     }
+
+    private boolean isOperationalData(String type) {
+        return !temporaryCache.contains(String.valueOf(type));
+    }
+
+    private String getSessionStoreQuery(String type) {
+        if (isOperationalData(type)) {
+            return sqlInsertSTORE;
+        } else {
+            return sqlInsertTempSTORE;
+        }
+    }
+
+    private String getSessionStoreDBQuery(String type, String query) {
+        if (isOperationalData(type)) {
+            return String.format(query, SESSION_STORE_TABLE);
+        } else {
+            return String.format(query, TEMP_SESSION_STORE_TABLE);
+        }
+    }
+
+    private String getSessionDeleteQuery(String type) {
+        if (isOperationalData(type)) {
+            return sqlInsertDELETE;
```

Comment:
Can we use string formatters for these two queries as well ? so that we don't need two queries defined ?. Is it possible ?